src/broker/cas_net_buf.h

```diff
@@ -88,6 +88,8 @@
 #define NET_BUF_TYPE_SIZE(net_buf) (DOES_CLIENT_UNDERSTAND_THE_PROTOCOL \
 ((net_buf)->client_version, PROTOCOL_V7) ? 2 * NET_SIZE_BYTE : NET_SIZE_BYTE)
 
+#define INT_DEFAULE_NET_BUF_SIZE        (16 * 1024)
```

Comment:
How about changing NET_BUF_KBYTE to 1024?

Reply:
For the readability, it is good to change 1024 to NET_BUF_KBYTE.
Changed.